tsconfig.json

```diff
@@ -12,8 +12,8 @@
     "forceConsistentCasingInFileNames": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noImplicitAny": true,
-    "strictNullChecks": true,
+    "noImplicitAny": false,
```

Comment:
Should we re-enable these?

Reply:
Erros are fired from jsonforms side, so at this point we can keep it like this